apps/frontend/src/renderer/components/task-detail/TaskMergedChanges.tsx

```
@@ -0,0 +1,383 @@
+          </h4>
+          <ScrollArea className="max-h-48">
+            <div className="space-y-1">
+              {commits.map((commit: MergedCommit) => (
+                <div
+                  key={commit.hash}
+                  className="group rounded-md border border-border/50 bg-muted/30 hover:bg-muted/50 transition-colors"
+                >
+                  <button
+                    onClick={() => toggleCommit(commit.hash)}
+                    className="w-full text-left px-3 py-2 flex items-start gap-2"
+                  >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)